app/main.py

```
@@ -1,52 +1,37 @@
-        elif serialize_type == "xml":
-            root = ET.Element("book")
-            title = ET.SubElement(root, "title")
-            title.text = self.title
-            content = ET.SubElement(root, "content")
-            content.text = self.content
-            return ET.tostring(root, encoding="unicode")
-        else:
-            raise ValueError(f"Unknown serialize type: {serialize_type}")
+from app.representation import ConsoleRepresentation, ReverseRepresentation
+from app.serializers import XMLSerializer, JsonSerializer
+from app.book import Book
 
 
 def main(book: Book, commands: list[tuple[str, str]]) -> None | str:
     for cmd, method_type in commands:
         if cmd == "display":
-            book.display(method_type)
+            if method_type == "reverse":
+                reverser = ReverseRepresentation()
+                reverser.display(book)
+            elif method_type == "console":
+                console = ConsoleRepresentation()
+                console.display(book)
+            else:
+                raise ValueError(f"Unknown display type: {method_type}")
         elif cmd == "print":
-            book.print_book(method_type)
+            if method_type == "reverse":
+                reverser = ReverseRepresentation()
+                reverser.print_book(book)
+            elif method_type == "console":
+                console = ConsoleRepresentation()
+                console.print_book(book)
+            else:
+                raise ValueError(f"Unknown print type: {method_type}")
         elif cmd == "serialize":
-            return book.serialize(method_type)
+            if method_type == "json":
+                serializer = JsonSerializer()
+                return serializer.serialize(book)
+            elif method_type == "xml":
+                serializer = XMLSerializer()
+                return serializer.serialize(book)
+            else:
+                ValueError(f"Unknown serialize type: {method_type}")
```

There is a missing `raise` keyword before `ValueError` in this line. It should be `raise ValueError(f"Unknown serialize type: {method_type}")` to properly raise the exception.